src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -156,50 +142,48 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
         if (distributeToRecipients) {
             String optionFragment =
                     FeedbackQuestionFormTemplates.populateTemplate(optionFragmentTemplate,
-                            "${qnIdx}", Integer.toString(qnIdx),
-                            "${responseIdx}", Integer.toString(responseIdx),
-                            "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled",
-                            "${constSumOptionVisibility}", "style=\"display:none\"",
-                            "${constSumOptionPoint}", existingConstSumResponse.getAnswerString(),
-                            "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}", Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
-                            "${constSumOptionValue}", "");
+                            Const.FeedbackQuestion.QUESTION_INDEX, Integer.toString(qnIdx),
```

Comment:
Let's change the value "${qnIndex} in FeedbackRubricQuestionDetails and the templates in this PR then. Perhaps we can prefix the constant names or put it in another inner class to make it clearer that these values are used only for templating?